framework/platform/bpl/uci/dhcp/bpl_dhcp.cpp

```diff
@@ -96,24 +100,35 @@ static int dhcp_event_handler(struct ubus_context *ctx, struct ubus_object *obj,
     return 0;
 }
 
-static const struct ubus_method dhcp_ubus_methods[] = {
-    UBUS_METHOD("dhcp_event", dhcp_event_handler, dhcp_event_policy),
-};
+static const struct ubus_method dhcp_ubus_methods[] = {{.name     = "dhcp_event",
```

Comment:
same question - shouldn't this be a separate commit?

Reply:
It's a tiny C/C++ build issue fix. Not critical.

Reply:
Why do you think it's a huge and wrong change?